FirebaseAuthUI/FUIAuthBaseViewController.h

```diff
@@ -80,6 +85,39 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)cancelAuthorization;
 
+/** @fn showSignInAlertWithEmail:provider:handler:
+ @brief Displays an alert to conform with user whether she wants to proceed with the provider.
```

Comment:
nit: `Displays an alert asking the user to confirm whether or not they want to proceed with the selected provider.`